docs/json_schemas/vault_kv/v0/requirer.json

```diff
@@ -6,10 +6,26 @@
           "description": "Suffix to append to the mount name to get the KV mount.",
           "title": "Mount Suffix",
           "type": "string"
+        },
+        "wrap_ttl": {
+          "anyOf": [
+            {
+              "type": "integer"
+            },
+            {
+              "type": "string"
+            },
+            {
+              "type": "null"
+            }
+          ],
```

Comment:
Why not force the type? Is there any value in providing options? I would encourage to select 1 and stick with it.

Reply:
[This is what Vault accepts](https://developer.hashicorp.com/vault/docs/concepts/response-wrapping#response-wrapping-token-creation). Either seconds as an int, or a string duration ("15s", "20m"). That doesn't necessarily mean we can't restrict it, but I can see the value and mimicking the underlying Vault parameter's accepted values.

Reply:
Yeah but Vault at least will return an error message if the value is not correct (Ex. `15bananas`). With charm relation interfaces we can't do this and there won't be any feedback to the KV requirer that what they provided was bad. I would encourage being quite restrictive here to avoid this category of problems.

Reply:
If this is already agreed and decided (based on your comments on the thread) I have no problem to turn it into an integer. Are we good to proceed with that change then?

Reply:
We discussed this comment as a team and the best approach is to define this interface to use only integers representing seconds. The library could enable passing different kinds of values and do the necessary calculations and error handling if we want, but it will make the interface more robust.

Reply:
please take a look at the updated version of the PR and let me know :slightly_smiling_face: